tests/example/flights2/flights2.dl

```diff
@@ -0,0 +1,49 @@
+.type Airport
```

Comment:
Needs the standard license header. e.g. https://github.com/souffle-lang/souffle/blob/master/tests/semantic/load7/load7.dl